src/NUnitEngine/Addins/nunit-v2-result-writer/NUnit2ResultSummary.cs

```diff
@@ -33,20 +32,20 @@ namespace NUnit.Engine.Addins
     /// </summary>
     public class NUnit2ResultSummary
     {
-        private int resultCount = 0;
-        private int testsRun = 0;
-        private int failureCount = 0;
-        private int errorCount = 0;
-        private int successCount = 0;
-        private int inconclusiveCount = 0;
-        private int skipCount = 0;
-        private int ignoreCount = 0;
-        private int notRunnable = 0;
-
-        private DateTime startTime = DateTime.MinValue;
-        private DateTime endTime = DateTime.MaxValue;
-        private double duration = 0.0d;
-        private string name;
```

Comment:
The initialization does no harm and makes it clear _when_ the values get initialized.